test/support/fake_scheduled_stop_time_data.ex

```diff
@@ -0,0 +1,27 @@
+defmodule FakeScheduledStopTimeData do
+    def create_fake_stop_time_file(file_name) do
+      fake_stop_file = get_fake_data_from_file(file_name)
+      case fake_stop_file do
+        {:ok, fake_stop_time_data} ->
+          fake_stop_time_data
+        {:error, message} ->
+          message |> IO.inspect
+      end
+      File.close(fake_stop_file)
+    end
+
+    defp get_fake_data_from_file(file_name) do
+      fake_stop_file = File.open(file_name, [:write],
+        fn file -> fake_raw_stop_times() |>
```

Comment:
Typically you put `|>` on the next line unless everything fits on a single line.